productMods/templates/freemarker/edit/forms/addProjectUpdate.ftl

```diff
@@ -180,6 +185,39 @@ Set this flag on the input acUriReceiver where you would like this behavior to o
     form#addProjectUpdate p {
         text-indent: 0;
     }
+    .HelperInfo {
+        position:relative;
+    }
+    .HelperInfo div {
+        display: none;
+    }
+    .HelperInfoHover {
+        position:relative;
+    }
+    .HelperInfoHover div {
+        display:block;
+        position:absolute;
+        width: 20em;
+        height: 6em;
+        bottom: 1em;
+        left: 1em;
+        z-index:1000;
+        background-color:#FFFFFF;
+        padding: 5px;
+        border-radius: 4px;
+        border-top-color: #5C85D6;
```

Comment:
these border styles were already defined by parent styles so needed to override them.